src/kube-runtime/deploy/runtime-exit-spec.yaml.template

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+{% for spec_item in cluster_cfg['k8s-job-exit-spec']['spec'] %}
+{%- if spec_item['code'] > 0 %}
+- containerExitCode: {{ spec_item['code'] }}
+{%- if 'reason' in spec_item %}
+  reason: {{ spec_item['reason'] }}
+{%- endif %}
+{%- if 'solution' in spec_item %}
+  solution:
+  {%- for str_val in spec_item['solution'] %}
+    - "{{ str_val }}"
+  {%- endfor %}
+{%- endif %}
```

Comment:
Why runtime need to read static reason, solution?

Reply:
Will remove these fields

---
In reply to: [322045079](https://github.com/microsoft/pai/pull/3585#discussion_r322045079) [](ancestors = 322045079)